pkg/persistence/mocks/backup_store.go

```diff
@@ -1,12 +1,32 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
+/*
+Copyright 2020 the Velero contributors.
```

Comment:
Thanks for moving this file in. Since I don't think my describe PR's going to use it, I'll just remove it from mine to prevent any merge conflicts.